src/main/java/frc/robot/subsystems/BallIndicator.java

```diff
@@ -125,13 +164,16 @@ private void updateStrip() {
 
   /* Run every robot loop. Checks whether to toggle the flashing LEDs */
   public void checkTimer() {
-    if (mStripTimer.get() >= 250) {
+    if (mStripTimer.get() % 250 == 0) {
```

Comment:
A comment as to why "% 250" replaces ">=250" would be nice

Reply:
I think this is designed to run every 25 ticks (250 milliseconds / 10 ms per tick) but it won't work because timers return values in seconds (I think) and the value won't line up perfectly on a 0 every time, resulting in the operation not equaling exactly 0 every time.

I'd recommend instead checking if the timer is >= 250 and then resetting and starting it.